ci/docker/install/ubuntu_core.sh

```diff
@@ -21,6 +21,7 @@
 # the whole docker cache for the image
 
 set -ex
+perl -pi -e 's/archive.ubuntu.com/us-west-2.ec2.archive.ubuntu.com/' /etc/apt/sources.list
```

Comment:
What's this going to do?

Reply:
use the AWS mirror instead of the default ubuntu one, making docker rebuild significantly faster.

Reply:
Can you make this conditional?